packages/cli-link-assets/src/linkAssets.ts

```diff
@@ -0,0 +1,227 @@
+        },
+        ios: {
+          isFontAsset: false,
+        },
+      };
+
+      result[imageType] = baseImageOptions;
+      return result;
+    },
+    {},
+  );
+
+  const audioLinkOptions = audioTypes.reduce(
+    (result: LinkOptionsPerExt, audioType) => {
+      const baseAudioOptions: LinkOptions = {
+        android: {
+          path: path.resolve(
+            androidPath,
+            androidAppName,
+            'src',
+            'main',
+            'res',
+            'raw',
+          ),
+          shouldUseFontXMLFiles: false,
+        },
+        ios: {
+          isFontAsset: false,
+        },
+      };
+
+      result[audioType] = baseAudioOptions;
+      return result;
+    },
+    {},
+  );
+
+  const linkOptionsPerExt: LinkOptionsPerExt = {
+    ...fontLinkOptions,
+    ...imageLinkOptions,
+    ...audioLinkOptions,
+  };
+
+  const customLinkOptions: LinkOptions = {
+    android: {
+      path: path.resolve(androidPath, 'app', 'src', 'main', 'assets', 'custom'),
+      shouldUseFontXMLFiles: false,
+    },
+    ios: {
+      isFontAsset: false,
+    },
+  };
+
+  const linkPlatformOptions: LinkPlatformOptions[] = [
+    {
+      name: 'Android',
+      enabled: true,
+      platform: 'android',
+      rootPath,
+      assetsPaths: androidAssetsPath,
+      manifest: getManifest(androidPath, 'android'),
+      platformConfig: {
+        exists: !!ctx.project.android,
+        path: androidPath,
+      },
+      cleanAssets: cleanAndroidAssets,
+      copyAssets: copyAndroidAssets,
+      linkOptionsPerExt: {
+        otf: linkOptionsPerExt?.otf?.android,
+        ttf: linkOptionsPerExt?.ttf?.android,
+        png: linkOptionsPerExt?.png?.android,
+        jpg: linkOptionsPerExt?.jpg?.android,
+        gif: linkOptionsPerExt?.gif?.android,
+        mp3: linkOptionsPerExt?.mp3?.android,
+      },
+      otherLinkOptions: customLinkOptions.android,
+    },
+    {
+      name: 'iOS',
+      enabled: true,
+      platform: 'ios',
+      rootPath,
+      assetsPaths: iosAssetsPath,
+      manifest: getManifest(iosPath, 'ios'),
+      platformConfig: {
+        exists: !!ctx.project.ios,
+        path: iosPath,
+        pbxprojFilePath: iosPbxprojFilePath,
+      },
+      cleanAssets: cleanIOSAssets,
+      copyAssets: copyIOSAssets,
+      linkOptionsPerExt: {
+        otf: linkOptionsPerExt?.otf?.ios,
+        ttf: linkOptionsPerExt?.ttf?.ios,
+        mp3: linkOptionsPerExt?.mp3?.ios,
+      },
+      otherLinkOptions: customLinkOptions.ios,
+    },
+  ];
+
+  linkPlatformOptions
+    .filter(({enabled, platformConfig}) => enabled && platformConfig.exists)
+    .forEach(linkPlatform);
+}
+
+export default {
+  func: linkAssets,
+  name: 'link-assets',
+  description: 'TODO',
+  options: [
+    {
+      name: '--project-root <string>',
+      description:
+        'Root path to your React Native project. When not specified, defaults to current working directory.',
+      default: process.cwd(),
+    },
+  ],
```

Comment:
I don't think this is necessary, project root is stored inside `Config`.

Reply:
In the original repo, these were the available options:

```
-p, --path - path to project, defaults to cwd.
-a, --assets - assets paths, for example react-native-asset -a ./src/font ./src/mp3.
-ios-a, --ios-assets - ios assets paths, will disable android linking
-android-a, --android-assets - android assets paths, will disable ios linking.
-n-u, --no-unlink - Not to unlink assets which not longer exists, not recommanded.
```

So maybe we don't need any parameters at all? All of them can be configured in `react-native-config.js` now, I kept `--project-root` to be a replacement for `--path`, but I agree with you that we could remove it too. 

I also removed `--no-unlink`, not sure if it's something really used.

WDYT @szymonrybczak @TMisiukiewicz ?

Reply:
Yeah, let's keep everything in `react-native-config.js`.